src/components/select/filterable-select/filterable-select.stories.mdx

```diff
@@ -192,6 +192,50 @@ import { FilterableSelect, Option } from "carbon-react/lib/components/select";
   </Story>
 </Preview>
 
+### With multiple columns:
+
+<Preview>
+  <Story
+    name="withMultipleColumns"
```

Comment:
This should be `name="with multiple columns"` really saying as it's displayed in the docs (I know some of the others have been done with camelCase, but they shouldn't have)